doc/development/submitting/submitting.md

```diff
@@ -22,38 +22,60 @@ Be sure to check your code against these rules:
 ### Use pre-commit
 
 It is highly recommended to install and use [Pre-commit](https://pre-commit.com)
```

Comment:
The pre-commit project seems to always use all-lowercase. Beginning of the sentence may be treated differently.